cmake/sca/eclair/sca.cmake

```diff
@@ -72,3 +82,13 @@ add_custom_command(
 )
 
 unset(ECLAIR_RULES_SET CACHE)
+unset(ECLAIR_metrics_tab CACHE)
+unset(ECLAIR_reports_tab CACHE)
+unset(ECLAIR_summary_txt CACHE)
+unset(ECLAIR_summary_doc CACHE)
+unset(ECLAIR_summary_odt CACHE)
+unset(ECLAIR_full_txt_areas CACHE)
+unset(ECLAIR_full_txt CACHE)
+unset(ECLAIR_full_doc_areas CACHE)
+unset(ECLAIR_full_doc CACHE)
+unset(ECLAIR_full_odt CACHE)
```

Comment:
Unset eventually externally provided variables that alter the behavior of the tool.
Just to avoid unexpected behaviors from the tool caused by previous runs settings.